monitoring/README.md

```diff
@@ -4,38 +4,67 @@
 
```

Comment:
I think it would be good to use the regular terminology for the two status add-on components. See https://docs.kubestellar.io/release-0.24.0/direct/architecture/ . One is called the status add-on controller and the other is the status add-on agent.

Reply:
Fixed - updated the architecture image to use that terminology.